[49.000024796] 2018-01-20 15:32:39,797 - INFO - launcher: Starting sanity checks
[50.000190735] 2018-01-20 15:32:39,798 - INFO - launcher: Checking for Python 3.5+
[50.000190735] 2018-01-20 15:32:39,798 - INFO - launcher: Checking console encoding
[50.000190735] 2018-01-20 15:32:39,798 - INFO - launcher: Setting console encoding to UTF-8
[50.000190735] 2018-01-20 15:32:39,798 - INFO - launcher: Ensuring we're in the right environment
[52.000045776] 2018-01-20 15:32:39,800 - INFO - launcher: Adding local bins/ folder to path
[53.000211716] 2018-01-20 15:32:39,801 - INFO - launcher: Required checks passed.
[53.000211716] 2018-01-20 15:32:39,801 - INFO - launcher: Optional checks passed.
[54.000139236] 2018-01-20 15:32:39,802 - INFO - launcher: Moving old musicbot log

######################### PRE-RUN SANITY CHECKS PASSED #########################

[1.6670000553131104] 2018-01-20 15:32:41,415 - DEBUG - musicbot.config | In config.py::MainThread(10284), line 234 in find_autoplaylist: Copying _autoplaylist.txt to autoplaylist.txt
[1.6670000553131104] 2018-01-20 15:32:41,415 - INFO - musicbot.permissions | In permissions.py::MainThread(10284), line 34 in __init__: Permissions file not found, copying example_permissions.ini
[2.5650002956390381] 2018-01-20 15:32:42,313 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 213 in _setup_logging: Set logging level to INFO
[2.5660002231597900] 2018-01-20 15:32:42,314 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 80 in __init__: =========== MusicBot (version 1.9.6_2) ===========
[2.5660002231597900] 2018-01-20 15:32:42,314 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 86 in __init__: Loaded autoplaylist with 2531 entries
[2.569000006] launcher-INFO: Connecting
[5.3359999656677246] 2018-01-20 15:32:45,084 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 1011 in on_ready: Connection established, ready to go.
[5.3359999656677246] 2018-01-20 15:32:45,084 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 324 in _cache_app_info: Caching app info
[5.4670000076293945] 2018-01-20 15:32:45,215 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 828 in _scheck_ensure_env: Ensuring data folders exist
[5.4700000286102295] 2018-01-20 15:32:45,218 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 844 in _scheck_server_permissions: Checking server permissions
[5.4700000286102295] 2018-01-20 15:32:45,218 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 848 in _scheck_autoplaylist: Auditing autoplaylist
[5.4709999561309814] 2018-01-20 15:32:45,219 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 852 in _scheck_configs: Validating config
[5.4709999561309814] 2018-01-20 15:32:45,219 - DEBUG - musicbot.config | In config.py::MainThread(10284), line 155 in async_validate: Validating options...
[5.4709999561309814] 2018-01-20 15:32:45,219 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 855 in _scheck_configs: Validating permissions config
[5.4719998836517334] 2018-01-20 15:32:45,220 - DEBUG - musicbot.permissions | In permissions.py::MainThread(10284), line 59 in async_validate: Validating permissions...
[5.4719998836517334] 2018-01-20 15:32:45,220 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 1022 in on_ready: Connected to Discord!
[5.4730000495910645] 2018-01-20 15:32:45,221 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 1032 in on_ready: Bot:   404275076624547860/Neko-Chan#5251 [BOT]
[5.4740002155303955] 2018-01-20 15:32:45,222 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 1040 in on_ready: Owner: 96564946304794624/Silver#2690

[5.4750001430511475] 2018-01-20 15:32:45,223 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 1043 in on_ready: Server List:
[5.4760000705718994] 2018-01-20 15:32:45,224 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 1044 in <listcomp>:  - Silberne Army
[5.4780001640319824] 2018-01-20 15:32:45,226 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 1088 in on_ready: Not bound to any text channels
[5.4790000915527344] 2018-01-20 15:32:45,227 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 1114 in on_ready: Not autojoining any voice channels
[5.4809999465942383] 2018-01-20 15:32:45,229 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 1118 in on_ready: Options:
[5.4820001125335693] 2018-01-20 15:32:45,230 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 1120 in on_ready:   Command prefix: !
[5.4839999675750732] 2018-01-20 15:32:45,232 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 1121 in on_ready:   Default volume: 25%
[5.4869999885559082] 2018-01-20 15:32:45,235 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 1123 in on_ready:   Skip threshold: 4 votes or 50%
[5.4900000095367432] 2018-01-20 15:32:45,238 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 1124 in on_ready:   Now Playing @mentions: Disabled
[5.4930002689361572] 2018-01-20 15:32:45,241 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 1125 in on_ready:   Auto-Summon: Enabled
[5.4960000514984131] 2018-01-20 15:32:45,244 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 1126 in on_ready:   Auto-Playlist: Enabled
[5.4980001449584961] 2018-01-20 15:32:45,246 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 1127 in on_ready:   Auto-Pause: Enabled
[5.5000000000000000] 2018-01-20 15:32:45,248 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 1128 in on_ready:   Delete Messages: Enabled
[5.5020000934600830] 2018-01-20 15:32:45,250 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 1130 in on_ready:     Delete Invoking: Disabled
[5.5039999485015869] 2018-01-20 15:32:45,252 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 1131 in on_ready:   Debug Mode: Disabled
[5.5049998760223389] 2018-01-20 15:32:45,253 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 1132 in on_ready:   Downloaded songs will be saved
[24.1800000667572021] 2018-01-20 15:33:03,928 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 2395 in on_message: 96564946304794624/Silver#2690: !help
[26.6540000438690186] 2018-01-20 15:33:06,402 - VOICEDEBUG - musicbot.bot | In bot.py::MainThread(10284), line 2564 in on_voice_state_update: Voice state update for 96564946304794624/Silver#2690 on Silberne Army/Laberecke 1 -> [CONNECT]
[36.5460002422332764] 2018-01-20 15:33:16,294 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 2395 in on_message: 96564946304794624/Silver#2690: !play
[36.5480000972747803] 2018-01-20 15:33:16,296 - ERROR - musicbot.bot | In bot.py::MainThread(10284), line 2515 in on_message: Error in play: CommandError: The bot is not in a voice channel.  Use !summon to summon it to your voice channel.
Traceback (most recent call last):
  File "D:\Wichtiges\Discord und Ts designs\MusicBot-master\MusicBot\musicbot\bot.py", line 2423, in on_message
    handler_kwargs['player'] = await self.get_player(message.channel)
  File "D:\Wichtiges\Discord und Ts designs\MusicBot-master\MusicBot\musicbot\bot.py", line 563, in get_player
    'Use %ssummon to summon it to your voice channel.' % self.config.command_prefix)
musicbot.exceptions.CommandError: The bot is not in a voice channel.  Use !summon to summon it to your voice channel.
[45.4960000514984131] 2018-01-20 15:33:25,244 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 2395 in on_message: 96564946304794624/Silver#2690: !summon
[45.4980001449584961] 2018-01-20 15:33:25,246 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 1804 in cmd_summon: Joining Silberne Army/Laberecke 1
[45.5000000000000000] 2018-01-20 15:33:25,248 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 437 in get_voice_client: Connection attempt 1 to Laberecke 1
[45.5000000000000000] 2018-01-20 15:33:25,248 - VOICEDEBUG - musicbot.bot | In bot.py::MainThread(10284), line 376 in join_voice_channel: (join_voice_channel) creating futures
[45.5010001659393311] 2018-01-20 15:33:25,249 - VOICEDEBUG - musicbot.bot | In bot.py::MainThread(10284), line 382 in join_voice_channel: (join_voice_channel) setting voice state
[45.5020000934600830] 2018-01-20 15:33:25,250 - VOICEDEBUG - musicbot.bot | In bot.py::MainThread(10284), line 385 in join_voice_channel: (join_voice_channel) waiting for session id
[45.6300001144409180] 2018-01-20 15:33:25,378 - VOICEDEBUG - musicbot.bot | In bot.py::MainThread(10284), line 2564 in on_voice_state_update: Voice state update for 404275076624547860/Neko-Chan#5251 on Silberne Army/Laberecke 1 -> [JOIN, CONNECT]
[45.6310000419616699] 2018-01-20 15:33:25,379 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 2575 in on_voice_state_update: 404275076624547860/Neko-Chan#5251 has joined Silberne Army/Laberecke 1
[45.6319999694824219] 2018-01-20 15:33:25,380 - VOICEDEBUG - musicbot.bot | In bot.py::MainThread(10284), line 389 in join_voice_channel: (join_voice_channel) waiting for voice data
[45.7320001125335693] 2018-01-20 15:33:25,480 - VOICEDEBUG - musicbot.bot | In bot.py::MainThread(10284), line 403 in join_voice_channel: (join_voice_channel) connecting...
[45.9250001907348633] 2018-01-20 15:33:25,673 - VOICEDEBUG - musicbot.bot | In bot.py::MainThread(10284), line 415 in join_voice_channel: (join_voice_channel) connection successful
[45.9260001182556152] 2018-01-20 15:33:25,674 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 463 in get_voice_client: Connected in 0.4s
[45.9260001182556152] 2018-01-20 15:33:25,674 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 464 in get_voice_client: Connected to Silberne Army/Laberecke 1
[45.9470000267028809] 2018-01-20 15:33:25,695 - VOICEDEBUG - musicbot.player | In player.py::MainThread(10284), line 311 in websocket_check: Starting websocket check loop for Silberne Army
[48.8519999980926514] 2018-01-20 15:33:28,600 - INFO - musicbot.entry | In entry.py::MainThread(10284), line 219 in _really_download: Download started: https://youtube.com/watch?v=0CoZG3S_JzA
[49.1340000629425049] 2018-01-20 15:33:28,882 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 2395 in on_message: 96564946304794624/Silver#2690: !play penis
[52.3180000782012939] 2018-01-20 15:33:32,066 - INFO - musicbot.entry | In entry.py::MainThread(10284), line 229 in _really_download: Download complete: https://youtube.com/watch?v=0CoZG3S_JzA
[52.3229999542236328] 2018-01-20 15:33:32,071 - FFMPEG - musicbot.player | In player.py::MainThread(10284), line 268 in _play: Creating player with options: -nostdin -vn audio_cache\youtube-0CoZG3S_JzA-bignic_-_Book_Of_Days.webm
[53.8080000877380371] 2018-01-20 15:33:33,556 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 781 in serialize_queue: Serializing queue for 404213840197320714
[53.8199999332427979] 2018-01-20 15:33:33,568 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 781 in serialize_queue: Serializing queue for 404213840197320714
[53.8229999542236328] 2018-01-20 15:33:33,571 - INFO - musicbot.entry | In entry.py::MainThread(10284), line 219 in _really_download: Download started: https://www.youtube.com/watch?v=cdErdEk56fs
[57.8389999866485596] 2018-01-20 15:33:37,587 - INFO - musicbot.entry | In entry.py::MainThread(10284), line 229 in _really_download: Download complete: https://www.youtube.com/watch?v=cdErdEk56fs
[61.0680000782012939] 2018-01-20 15:33:40,816 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 2395 in on_message: 96564946304794624/Silver#2690: !skip
[61.0910000801086426] 2018-01-20 15:33:40,839 - FFMPEG - musicbot.player | In player.py::MainThread(10284), line 268 in _play: Creating player with options: -nostdin -vn audio_cache\youtube-cdErdEk56fs-The_Man_With_The_Half-Metre_Penis.webm
[61.1570003032684326] 2018-01-20 15:33:40,905 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 781 in serialize_queue: Serializing queue for 404213840197320714
[61.1610000133514404] 2018-01-20 15:33:40,909 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 781 in serialize_queue: Serializing queue for 404213840197320714
[90.2050001621246338] 2018-01-20 15:34:09,953 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 2395 in on_message: 96564946304794624/Silver#2690: !clear
[114.0100002288818359] 2018-01-20 15:34:33,758 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 2395 in on_message: 96564946304794624/Silver#2690: !skip
[117.7239999771118164] 2018-01-20 15:34:37,472 - INFO - musicbot.entry | In entry.py::MainThread(10284), line 219 in _really_download: Download started: https://youtube.com/watch?v=I4keD5Qvya4
[121.6150000095367432] 2018-01-20 15:34:41,363 - INFO - musicbot.entry | In entry.py::MainThread(10284), line 229 in _really_download: Download complete: https://youtube.com/watch?v=I4keD5Qvya4
[121.6199998855590820] 2018-01-20 15:34:41,368 - FFMPEG - musicbot.player | In player.py::MainThread(10284), line 268 in _play: Creating player with options: -nostdin -vn audio_cache\youtube-I4keD5Qvya4-Thomas_Rhett_-_It_Goes_Like_This.webm
[121.6620001792907715] 2018-01-20 15:34:41,410 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 781 in serialize_queue: Serializing queue for 404213840197320714
[130.4110000133514404] 2018-01-20 15:34:50,159 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 2395 in on_message: 96564946304794624/Silver#2690: !disconnect
[130.5710000991821289] 2018-01-20 15:34:50,319 - VOICEDEBUG - musicbot.bot | In bot.py::MainThread(10284), line 2564 in on_voice_state_update: Voice state update for 404275076624547860/Neko-Chan#5251 on Silberne Army/Laberecke 1 -> [DISCONNECT]
[205.3700001239776611] 2018-01-20 15:36:05,118 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 2395 in on_message: 96564946304794624/Silver#2690: !help
[211.0039999485015869] 2018-01-20 15:36:10,752 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 2395 in on_message: 96564946304794624/Silver#2690: !clear
[211.0050001144409180] 2018-01-20 15:36:10,753 - ERROR - musicbot.bot | In bot.py::MainThread(10284), line 2515 in on_message: Error in clear: CommandError: The bot is not in a voice channel.  Use !summon to summon it to your voice channel.
Traceback (most recent call last):
  File "D:\Wichtiges\Discord und Ts designs\MusicBot-master\MusicBot\musicbot\bot.py", line 2423, in on_message
    handler_kwargs['player'] = await self.get_player(message.channel)
  File "D:\Wichtiges\Discord und Ts designs\MusicBot-master\MusicBot\musicbot\bot.py", line 563, in get_player
    'Use %ssummon to summon it to your voice channel.' % self.config.command_prefix)
musicbot.exceptions.CommandError: The bot is not in a voice channel.  Use !summon to summon it to your voice channel.
[213.5580000877380371] 2018-01-20 15:36:13,306 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 2395 in on_message: 96564946304794624/Silver#2690: !clean
[213.7570002079010010] 2018-01-20 15:36:13,505 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 906 in safe_delete_message: Cannot delete message "!clean", no permission
[216.9430000782012939] 2018-01-20 15:36:16,691 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 2395 in on_message: 96564946304794624/Silver#2690: !summon
[216.9490001201629639] 2018-01-20 15:36:16,697 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 1804 in cmd_summon: Joining Silberne Army/Laberecke 1
[216.9529998302459717] 2018-01-20 15:36:16,701 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 437 in get_voice_client: Connection attempt 1 to Laberecke 1
[216.9550001621246338] 2018-01-20 15:36:16,703 - VOICEDEBUG - musicbot.bot | In bot.py::MainThread(10284), line 376 in join_voice_channel: (join_voice_channel) creating futures
[216.9550001621246338] 2018-01-20 15:36:16,703 - VOICEDEBUG - musicbot.bot | In bot.py::MainThread(10284), line 382 in join_voice_channel: (join_voice_channel) setting voice state
[216.9560000896453857] 2018-01-20 15:36:16,704 - VOICEDEBUG - musicbot.bot | In bot.py::MainThread(10284), line 385 in join_voice_channel: (join_voice_channel) waiting for session id
[217.1260001659393311] 2018-01-20 15:36:16,874 - VOICEDEBUG - musicbot.bot | In bot.py::MainThread(10284), line 2564 in on_voice_state_update: Voice state update for 404275076624547860/Neko-Chan#5251 on Silberne Army/Laberecke 1 -> [JOIN, CONNECT]
[217.1270000934600830] 2018-01-20 15:36:16,875 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 2575 in on_voice_state_update: 404275076624547860/Neko-Chan#5251 has joined Silberne Army/Laberecke 1
[217.1359999179840088] 2018-01-20 15:36:16,884 - VOICEDEBUG - musicbot.bot | In bot.py::MainThread(10284), line 389 in join_voice_channel: (join_voice_channel) waiting for voice data
[217.2089998722076416] 2018-01-20 15:36:16,957 - VOICEDEBUG - musicbot.bot | In bot.py::MainThread(10284), line 403 in join_voice_channel: (join_voice_channel) connecting...
[217.7360000610351562] 2018-01-20 15:36:17,484 - VOICEDEBUG - musicbot.bot | In bot.py::MainThread(10284), line 415 in join_voice_channel: (join_voice_channel) connection successful
[217.7360000610351562] 2018-01-20 15:36:17,484 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 463 in get_voice_client: Connected in 0.8s
[217.7360000610351562] 2018-01-20 15:36:17,484 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 464 in get_voice_client: Connected to Silberne Army/Laberecke 1
[217.7380001544952393] 2018-01-20 15:36:17,486 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 805 in deserialize_queue: Deserializing queue for 404213840197320714
[218.2320001125335693] 2018-01-20 15:36:17,980 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 555 in get_player: Created player via deserialization for server 404213840197320714 with 1 entries
[218.2320001125335693] 2018-01-20 15:36:17,980 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 781 in serialize_queue: Serializing queue for 404213840197320714
[218.2330000400543213] 2018-01-20 15:36:17,981 - VOICEDEBUG - musicbot.player | In player.py::MainThread(10284), line 311 in websocket_check: Starting websocket check loop for Silberne Army
[218.2340002059936523] 2018-01-20 15:36:17,982 - FFMPEG - musicbot.player | In player.py::MainThread(10284), line 268 in _play: Creating player with options: -nostdin -vn audio_cache\youtube-I4keD5Qvya4-Thomas_Rhett_-_It_Goes_Like_This.webm
[218.2489998340606689] 2018-01-20 15:36:17,997 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 781 in serialize_queue: Serializing queue for 404213840197320714
[221.5940001010894775] 2018-01-20 15:36:21,342 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 2395 in on_message: 96564946304794624/Silver#2690: !clear
[224.1689999103546143] 2018-01-20 15:36:23,917 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 2395 in on_message: 96564946304794624/Silver#2690: !help
[265.8420000076293945] 2018-01-20 15:37:05,590 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 909 in safe_delete_message: Cannot delete message "@Silver, **Available commands**
```!blacklist, !clean, !clear, !disconnect, !id, !joinserver, !listids, !np, !pause, !perms, !play, !pldump, !queue, !restart, !resume, !save, !search, !setavatar, !setname, !setnick, !shuffle, !shutdown, !skip, !stream, !summon, !volume```
<https://github.com/SexualRhinoceros/MusicBot/wiki/Commands-list>You can also use `!help x` for more info about each command.", message not found
[414.9879999160766602] 2018-01-20 15:39:34,736 - INFO - musicbot.entry | In entry.py::MainThread(10284), line 219 in _really_download: Download started: https://youtube.com/watch?v=4hAz3oI-VD0
[418.1540000438690186] 2018-01-20 15:39:37,902 - INFO - musicbot.entry | In entry.py::MainThread(10284), line 229 in _really_download: Download complete: https://youtube.com/watch?v=4hAz3oI-VD0
[418.1570003032684326] 2018-01-20 15:39:37,905 - FFMPEG - musicbot.player | In player.py::MainThread(10284), line 268 in _play: Creating player with options: -nostdin -vn audio_cache\youtube-4hAz3oI-VD0-Shirobon_-_Out_Of_Love.webm
[418.1730000972747803] 2018-01-20 15:39:37,921 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 781 in serialize_queue: Serializing queue for 404213840197320714
[703.2599999904632568] 2018-01-20 15:44:23,008 - ERROR - musicbot.bot | In bot.py::MainThread(10284), line 656 in on_player_finished_playing: Error processing youtube url:
ERROR: BLR_wRa0hXo: YouTube said: Invalid parameters.
[703.2620000839233398] 2018-01-20 15:44:23,010 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 337 in remove_from_autoplaylist: Removing unplayable song from autoplaylist: https://youtube.com/watch?v=BLR_wRa0hXo
[703.2649998664855957] 2018-01-20 15:44:23,013 - INFO - musicbot.bot | In bot.py::MainThread(10284), line 351 in remove_from_autoplaylist: Updating autoplaylist
[706.5629999637603760] 2018-01-20 15:44:26,311 - INFO - musicbot.entry | In entry.py::MainThread(10284), line 219 in _really_download: Download started: https://youtube.com/watch?v=OyQVjGdJ60g
[710.0670001506805420] 2018-01-20 15:44:29,815 - INFO - musicbot.entry | In entry.py::MainThread(10284), line 229 in _really_download: Download complete: https://youtube.com/watch?v=OyQVjGdJ60g
[710.0720000267028809] 2018-01-20 15:44:29,820 - FFMPEG - musicbot.player | In player.py::MainThread(10284), line 268 in _play: Creating player with options: -nostdin -vn audio_cache\youtube-OyQVjGdJ60g-Lionel_Richie_All_night_Long.webm
[710.0940001010894775] 2018-01-20 15:44:29,842 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 781 in serialize_queue: Serializing queue for 404213840197320714
[972.8370001316070557] 2018-01-20 15:48:52,585 - INFO - musicbot.entry | In entry.py::MainThread(10284), line 219 in _really_download: Download started: https://youtube.com/watch?v=smJ617KKpcA
[977.1919999122619629] 2018-01-20 15:48:56,940 - INFO - musicbot.entry | In entry.py::MainThread(10284), line 229 in _really_download: Download complete: https://youtube.com/watch?v=smJ617KKpcA
[977.1990001201629639] 2018-01-20 15:48:56,947 - FFMPEG - musicbot.player | In player.py::MainThread(10284), line 268 in _play: Creating player with options: -nostdin -vn audio_cache\youtube-smJ617KKpcA-The_Oscar_Peterson_Trio_-_I_ve_Got_It_Bad_That_Ain_t_Good.webm
[977.2239999771118164] 2018-01-20 15:48:56,972 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 781 in serialize_queue: Serializing queue for 404213840197320714
[1285.8100001811981201] 2018-01-20 15:54:05,558 - INFO - musicbot.entry | In entry.py::MainThread(10284), line 219 in _really_download: Download started: https://youtube.com/watch?v=7BhYmwFSOas
[1290.4850001335144043] 2018-01-20 15:54:10,233 - INFO - musicbot.entry | In entry.py::MainThread(10284), line 229 in _really_download: Download complete: https://youtube.com/watch?v=7BhYmwFSOas
[1290.4910001754760742] 2018-01-20 15:54:10,239 - FFMPEG - musicbot.player | In player.py::MainThread(10284), line 268 in _play: Creating player with options: -nostdin -vn audio_cache\youtube-7BhYmwFSOas-Glitch_Hop_or_110BPM_-_Volant_-_Minty_Monstercat_Release.webm
[1290.5069999694824219] 2018-01-20 15:54:10,255 - DEBUG - musicbot.bot | In bot.py::MainThread(10284), line 781 in serialize_queue: Serializing queue for 404213840197320714
